src/ioctls/vm.rs

```diff
@@ -608,6 +608,15 @@ impl AsRawFd for VmFd {
     }
 }
 
+impl Clone for VmFd {
+    fn clone(&self) -> Self {
```

Comment:
nit: I know there is no TryClone trait available in the stable standard library, but did you consider instead manually implementing `try_clone()` instead as I think the error from underlying `try_clone()` could be recoverable.

Reply:
I agree with Rob here. I think it's better to avoid expects in libraries and let the consumer handle the error.

Reply:
Ok I have worked around the problem by using `Arc<>` from the code consuming the `VmFd`. There's no need for the library to handle the `VmFd` cloning.